src/utils/calculator.ts

```diff
@@ -0,0 +1,67 @@
+import { GatewayOracle } from './gateway_oracle';
+import type { ArweaveOracle } from './arweave_oracle';
+import regression, { DataPoint } from 'regression';
+
+const SET = 'SET';
+
+export class Calculator {
```

Comment:
Lets call this something different here. Like RegressionCalculator or something.

@jdaev has a `<Calculator />` component coming in from the PE-66 branch

Reply:
This was dealt with in the changes I pushed, namely dd85f70